code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -313,6 +170,10 @@ var/list/gaslist_cache = init_gaslist_cache()
 	//Compares sample to self to see if within acceptable ranges that group processing may be enabled
 	//Returns: a string indicating what check failed, or "" if check passes
 
+/datum/gas_mixture/proc/react(turf/open/dump_location)

```

Comment:
He didn't use mso's test, which randomizes the proc calls. What MSO has discovered is that the proc called second in these types of tests will be called (or executed?) faster, so if you swap the proc call order you get different results, so it's not a reliable test.